flash_cards.py

```diff
@@ -145,6 +146,9 @@
 
 1. Create a function called main(), have it take in no arguments
 
+An example can be found below. Make sure to delete it before you create your new main() function.
```

Comment:
I think asking them to create main() was not ideal. Maybe I just ask them to alter it?

Reply:
Yea let me change that